File: backend/app/compliance/rules/boot_sanitation_rule.py
from __future__ import annotations

from app.compliance.constants import COMPLIANCE_RULE_IDS
from app.compliance.rules.base import BaseComplianceRule
from app.compliance.types import ComplianceContext, ComplianceEvaluationResult


class BootSanitationRule(BaseComplianceRule):
    """Skeleton — v1.8 will implement boot disinfection workflow checks."""

    id = COMPLIANCE_RULE_IDS["NO_BOOT_SANITIZATION"]
    name = "Không sát trùng ủng"
    event_type = "NO_BOOT_SANITIZATION"

    def evaluate(self, context: ComplianceContext) -> ComplianceEvaluationResult:
        return ComplianceEvaluationResult(violated=False, score=0.0, evidence={})

File: backend/src/modules/compliance/rules/bootSanitationRule.js
import { COMPLIANCE_RULE_IDS } from '../complianceRules.js'

/** @type {import('../complianceRules.js').ComplianceRule} */
export const bootSanitationRule = {
  id: COMPLIANCE_RULE_IDS.NO_BOOT_SANITIZATION,
  name: 'Không sát trùng ủng',
  eventType: 'NO_BOOT_SANITIZATION',
  enabled: true,
  evaluate() {
    return { violated: false, score: 0, evidence: {} }
  },
}
